src/main/java/ru/spbstu/model/Annotation.java

```diff
@@ -0,0 +1,37 @@
+package ru.spbstu.model;
+
+import com.vladmihalcea.hibernate.type.json.JsonBinaryType;
+import lombok.Data;
+import lombok.experimental.Accessors;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import java.util.Map;
+
+@Data
+@Accessors(chain = true)
+@Table(name = "annotations")
+@TypeDef(name = "jsonb", typeClass = JsonBinaryType.class)
+@Entity
+public class Annotation {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+  @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
I suggest using `EAGER` fetching type because we will need to load `variant` for the most cases.
Eager loading is default for `@ManyToOne` relations so we can just remove `(fetch = FetchType.LAZY)`.

Reply:
i didnt know that we'll use it for the most cases. Changed to EAGER